### ISSUE TITLE
Prevent pages to be closed when creating a post / Draft feature request

### ISSUE THREAD

**Issue by tio-trom** [friendica/friendica#10476]
I just created a big post on Friendica in the "dedicated" post composer page. By mistake I closed the tab and lost it all...I got so sad that I gave up re-writing it again. It would help tremendously if you will prevent a page from closing unless users approve that, when a post is created/edited. This is a basic function wordpress and most websites have. I think it really is a must. Cheers!

**Comment by MrPetovan** [comment]
Sorry about that, did you try to restore your tab? Firefox, Chrome and Safari allow you to restore a closed tab including all the data in the form if any. This wouldn't work on mobile but closing tabs is a little bit more involved and is less easily doable by mistake.

I don't like the Javascript alert on page close, the ideal would be drafts but it is way more involved.

**Comment by tio-trom** [comment]
Yup I restored it but it did not restore the text. Would be really useful if all posts would be saved somehow as drafts or as said prevent the tab from closing unless you approve that. On Friendica you can create very long posts and format them very nicely similar to blog posts, so I think a draft feature that autosaves your post is very useful. Maybe keep this issue open till you guys get some help/time and implement a solution for this ;).

**Comment by MrPetovan** [comment]
Interesting, just tested closing/reopening the tab on Vivaldi on Windows and it restored the text I entered. And yeah, I thought there was an issue open for a draft feature but I was wrong, let's keep this for this purpose.
